zcash_primitives/Cargo.toml

```diff
@@ -63,9 +63,6 @@ hdwallet = { workspace = true, optional = true }
 #   - `SecretKey` and `PublicKey` types exposed
 secp256k1 = { workspace = true, optional = true }
 
-# - ZIP 339
-bip0039 = { version = "0.10", features = ["std", "all-languages"] }
```

Comment:
This removal results in changes to `Cargo.lock` that were not included in the commit. Please include those changes in the commit removing this dependency.

Reply:
Are you sure it should be in the same commit? That commit is a pure reversion of d1443e5049c32615155eaed0e36915dbe5222f6b.

Reply:
Given that it is a pure reversion, fine to be in a follow-on commit.